source/elements/oneMKL/source/domains/spblas/operations/spmm.rst

```diff
@@ -273,9 +273,19 @@ spmm
      before ``spmm`` with the same arguments. ``spmm`` can then be called
      multiple times. Calling ``spmm_optimize`` on the same descriptor can reset
      some of the descriptor's data such as the ``workspace``.
+   - In the general case, not calling the functions in the order specified above
+     is undefined behavior. Not calling ``spmm_buffer_size`` or
+     ``spmm_optimize`` at least once will throw an
```

Comment:
Maybe the following, since that's how you have also implemented it in https://github.com/oneapi-src/oneMKL/pull/500?
```suggestion
     ``spmm_optimize`` at least once with a given descriptor will throw an
```

Reply:
Done in https://github.com/uxlfoundation/oneAPI-spec/pull/542/commits/b3eb26b577e60ea089e47cd306468c194ce8bc63